docs/technology-guidelines/docker-compose.md

```diff
@@ -0,0 +1,103 @@
+# Docker Compose
+
+Docker compose is used for local development and in the [CI/CD pipeline](../cicd/index.md) to build docker containers and run tests.
+
+## Use override files to reduce duplication
+
+Additional settings can be applied to a docker compose file by using override files.
+
+One use case is for testing - common settings can be put into the base `docker-compose.yaml` file, while changes to the command and container names can be placed in override files.
```

Comment:
I would like to add something about the docker-compose.override.yaml file, as this could be confusing to a new developer who is unfamiliar with the way docker-compose treats these files.
The docker-compose.override.yaml file is only referenced by docker-compose when you don't specify -f option on the command line. 

So running `docker-compose up -d` will include the `docker-compose.override.yaml` file, but `docker-compose -f docker-compose.yaml` will not.

So we should make it clear that running `docker-compose -f docker-compose.yaml -f docker-compose.test.yaml` will not include the `docker-compose.override.yaml`

Reply:
That is a confusing thing for anyone new to Docker Compose. Would be good to include a link to the documentation on that particular behaviour: https://docs.docker.com/compose/reference/overview/#specifying-multiple-compose-files